desktop/appcontainer/Banner.scss

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+body.xh-app .xh-banner {
+  height: var(--xh-banner-height-px);
+
+  & > .xh-icon {
+    margin-right: var(--xh-pad-half-px);
+  }
+
+  &__message {
+    margin-right: var(--xh-pad-px);
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
+  &__dismiss-button {
+    color: white !important;
```

Comment:
I wasn't a super big fan of adding these `!important`s as I did here and below, but I really did want the button to stand out, and I didn't want to string together all the classnames otherwise required to override. Hope this + the `xh-intent-none` change seem reasonable.

Reply:
👍